pages/countries/index.js

```diff
@@ -20,24 +25,55 @@ export default function Countries({ surfspots }) {
     router.push(`/`);
   }
 
+  function handleChange(event) {
+    setSearchTerm(event.target.value);
+    let searchValue = event.target.value;
+    setSearchedSpots(
+      surfspots.filter((spot) => spot.slug.includes(searchValue.toLowerCase()))
+    );
+
+    if (event.target.value === "") {
+      setSearchTerm(false);
+    } else {
+      setSearchTerm(true);
+    }
+  }
+
   return (
     <>
       <Head>
         <title>SpotiFinder</title>
       </Head>
-      <Header onMapShown={handleMapShown} usedMapShown={false} />
+      <MenueHeader onMapShown={handleMapShown} usedMapShown={false} />
       <ListSurfspots>
-        <FavoriteSpotsListItem name={"Favoriten"} link={`/favoritespots`} />
-        {uniqueCountrys.map((uniqueCountry) => {
-          const countryName = capitalizeFirstLetter(uniqueCountry);
-          return (
-            <ListSurfspotsItems
-              link={`/countries/${uniqueCountry}`}
-              name={countryName}
-              key={uniqueCountry}
-            />
-          );
-        })}
+        <SearchSpot onChange={handleChange} />
+        {searchTerm ? (
+          <>
+            {searchedSpots.map((spot) => {
+              return (
+                <ListSurfspotsItems
+                  link={`/${spot.slug}`}
+                  name={spot.name}
+                  key={spot._id}
+                />
+              );
+            })}
+          </>
+        ) : (
+          <>
+            <FavoriteSpotsListItem name={"Favoriten"} link={`/favoritespots`} />
```

Comment:
```suggestion
            <FavoriteSpotsListItem name="Favoriten" link={`/favoritespots`} />
```
glaub die Klammern brauchst du nicht, wenn es nicht dynamisch ist